binary_transparency/firmware/cmd/ft_witness/README.md

```diff
@@ -0,0 +1,35 @@
+# STH Witness
+
+
+## Introduction
+
+Firmware Transparency Demo steps outlined under [Firmware Transparency](firmware) provides a robust facility against verifying that the supplied firmware update by a FW Vendor is visible via a public log server. The supplied proofs from the log server and its verification locally by clients give sufficient confidence that the FW is verifiable, and the FT log tree is consistent, as presented to individual clients by the log server.
```

Comment:
I would change "FW is verifiable" to "FW is discoverable".

Reply:
Done 👍